project_setup/scripts/terraform-to-secrets

```diff
@@ -207,9 +217,16 @@ def parse_creds(terraform_state: Dict) -> Generator[Tuple[str, str, str], None,
 
 def parse_tagged_resources(
     terraform_state: Dict,
+    include_remote_state: bool,
 ) -> Generator[Tuple[str, str], None, None]:
     """Search all resources for tags requesting the creation of a secret."""
     for resource in find_resources(terraform_state, None):
```

Comment:
Is there a reason we don't push remote state logic into `find_resources()` as we do for `find_outputs()`?

Reply:
What are your thoughts on moving the other check to `parse_tagged_outputs()` then? I'd just prefer consistency if it's not a big leap to do so since the functions are logically so similar.

Reply:
Unfortunately, the resource type is only available in `find_outputs()` and not in `parse_tagged_outputs()`, so that's why that check is located in `find_outputs()`.